third-party/linshare/src/test/resources/docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
         command: mongod --smallfiles
 
     smtp:
+        container_name: linshare_smtp
```

Comment:
This name may be also given through a link defined in linshare-backend IMO 

(allowing to run this dockerfile two time at the same time and avoids naming conflicts)